### DISCUSSION TITLE
Mobile friendly interface

### DISCUSSION THREAD

**Discussion by aimerib** [mcmonkeyprojects/SwarmUI#85] [Show and tell]
Hi all,
I've been a huge fan of SwarmUI since I've found it, and the biggest hurdle for me personally was the lack of mobile support. I don't want to inundate the repo with PRs if you guys already have this on your radar, but I went ahead and made a few changes to make swarmui more mobile friendly. It isn't perfect, and I would not ship this as a complete feature, but rather a starting point to make it easier for others. I have been using my branch daily now, and things work 98% of the time (with only iOS auto-resizing when selecting an input causing any minor inconveniences)

Here's a demo of SwarmUI running on an iPhone 11 Pro (pretty small screen by today's standards):
https://github.com/user-attachments/assets/1f678a40-ef46-4bb1-8303-bdd916f84c57

**Comment by mcmonkey4eva** [comment]
See here https://github.com/mcmonkeyprojects/SwarmUI/issues/39

If your fork works well, please make a PR
